umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/provisioning.md

```diff
@@ -1,8 +1,40 @@
 ---
 title: Provisioning
 menuTitle: Provisioning
-description:
+description: Learn the different ways to provision the the Data and the Device & Container Infrastructures.
 weight: 1000
 edition: community
-draft: true
+draft: false
 ---
+
+Using the UMH's Management Console, you can provision the UMH in a simple process.
```

Comment:
Using the Management Console, you can provision the United Manufacturing Hub in a simple process.